src/test/llmnode_test.cpp

```diff
@@ -145,6 +145,12 @@ TEST_F(LLMFlowHttpTest, unaryCompletionsJson) {
     }
     ASSERT_EQ(d["model"], "llmDummyKFS");
     ASSERT_EQ(d["object"], "text_completion");
+
+    ASSERT_TRUE(d["usage"].IsObject());
+    ASSERT_TRUE(d["usage"].GetObject()["prompt_tokens"].IsInt());
+    ASSERT_TRUE(d["usage"].GetObject()["completion_tokens"].IsInt());
+    ASSERT_TRUE(d["usage"].GetObject()["total_tokens"].IsInt());
+    ASSERT_EQ(d["usage"].GetObject()["completion_tokens"].GetInt(), 5 /* max_tokens */);
```

Comment:
can we really assert the number? the model output can change

Reply:
I think it's pretty safe. We set max_tokens to very low number and it's very unlikely that model will naturally stop generating after less than 5 tokens.